File: sorts/bubble_sort.py
# bubble sort

class Solution:
    def sortArray(self, nums: List[int]) -> List[int]:
        n = len(nums)
        for i in range(n):
            min_index = i
            for j in range(0, n - i - 1):
                if nums[j] > nums[j + 1]:
                    if nums[j] > nums[j + 1]:
                        nums[j], nums[j + 1] = nums[j + 1], nums[j]

        return nums

File: sorts/bubble_sort.rb
def sort_array(nums)
    n = nums.size
    return nums if n <= 1
    
    n.times do |i|
        (0...(n - i - 1)).each do |j|
            if nums[j] > nums[j + 1]
                nums[j], nums[j + 1] = nums[j + 1], nums[j]
            end
        end
    end
    
    nums
end